docs/learn/learn-nomination-pools.md

```diff
@@ -207,10 +224,62 @@ balance.
   all other members have left. Once they leave by withdrawing, the pool is fully removed from the
   system.
 - Nominator: Can select the validators the pool nominates.
-- State-Toggler: Can change the pool’s state and kick (permissionlessly unbond/withdraw) members if
-  the pool is blocked.
-- Root: Can change the nominator, state-toggler, or itself. Further, it can perform any of the
-  actions the nominator or state-toggler can.
+- Bouncer: Can change the pool’s state and kick (permissionlessly unbond/withdraw) members if the
+  pool is blocked.
+- Root: Can change the nominator, bouncer, or itself. Further, it can perform any of the actions the
+  nominator or bouncer can.
+
+### Pool Commissions
+
+:::info Live on Kusama
+
+Pool commissions are currently live on Kusama. On Polkadot will be live once the Max Commission will
+be decided through governance referendum.
+
+:::
+
+As a pool admin, you can set pool commissions that will be applied to the staking rewards paid out
+to the pool's system account before rewards are allocated for the pool members.
+
+Three methods can be used when setting the pool commission:
+
+- **Commission Rate** (`nominationPools.setCommission` extrinsic): the start or new commission rate
+  (`newCommission` parameter) that can be set between 0% and the Max Commission (decided through
+  [governance referendum](./learn-polkadot-opengov.md)). You will need to specify an Input Payee
+  Account, i.e. the account that will receive the commission.
+- **Max Commission** (`nominationPools.setCommissionMax` extrinsic): the maximum commission
+  (`maxCommission` parameter) the pool will apply to its members (between 0% and Max Commission).
+  Note that once set, **the pool admin can only lower it**.
+- **Change Rate** (`nominationPools.setCommissionChangeRate` extrinsic): the maximum rate increase
+  (`maxIncrease` parameter) allowed for a single commission update. Note that once set, **the pool
+  admin can only lower it**. When setting the Change Rate, it will also be possible to set a
+  `minDelay` quantified as the number of blocks (since last commission update) after which it is
+  possible to change the commission (i.e. the minimum delay between commission updates). Note that
+  once set, **the pool admin can only increase it**.
+
+Max Commission and Change Rate must not be necessarily set. It is the choice of the pool admin to
+set those parameters and provide transparency to the pool members about the pool's commission
+policy.
+
+:::warning Max Commission and Change Rate are permanent
+
+Once the Max Commission and the Change Rate are set, the pool admin can only decrease those values.
+The minimum delay between commission updates can only be increased.
+
+:::
+
+Let's take, for example, Pool A, which sets the Commission Rate to 10%, the Max Commission to 100%,
+and the Change Rate to 1% every 30 minutes. The following statements are true:
```

Comment:
Excellent explanation 👍

One nit, perhaps say the number of blocks here (as this is what is stored on chain), and then "which equates to approximately 30 minutes".